apps/frontend/src/app/PageTeam/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -224,16 +224,16 @@ export default function TabBuild() {
     [database, loadoutData, teamCharId]
   )
   const filteredArts = useMemo(() => {
-    const {
-      mainStatKeys,
-      excludedLocations,
-      artExclusion,
-      levelLow,
-      levelHigh,
-      allowLocationsState,
-      useExcludedArts,
-      useTeammateBuild,
-    } = deferredArtsDirty && deferredBuildSetting
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
Can you move the object deconstruction above this function? seems to be a lot of boilerplate doing it this way.